2023-05-22 17:44:17,733 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:44:17,891 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:44:18,052 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:44:18,103 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:44:18,252 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:44:18,314 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:44:18,521 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:44:18,572 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:44:18,756 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:44:18,806 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:44:18,919 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:44:18,969 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:44:19,083 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:44:19,133 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:44:19,317 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:44:19,367 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:44:19,532 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:44:19,591 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:44:19,830 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:44:19,880 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:44:20,019 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:45:17,991 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:45:18,164 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:45:18,353 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:45:18,409 INFO scheduler Scheduled Job Start: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 17:45:18,622 INFO scheduler Scheduled Job Complete: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 17:45:18,678 INFO scheduler Scheduled Job Start: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 17:45:18,903 INFO scheduler Scheduled Job Complete: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 17:45:18,958 INFO scheduler Scheduled Job Start: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 17:45:19,174 INFO scheduler Scheduled Job Complete: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 17:46:18,244 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:47:18,481 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:48:18,716 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:48:18,859 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:48:19,016 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:48:19,067 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:48:19,215 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:48:19,268 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:48:19,397 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:48:19,447 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:48:19,631 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:48:19,680 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:48:19,793 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:48:19,842 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:48:19,955 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:48:20,007 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:48:20,192 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:48:20,242 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:48:20,402 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:48:20,452 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:48:20,625 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:48:20,675 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:48:20,799 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:49:18,971 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:50:19,242 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:50:19,430 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:50:19,672 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:51:19,506 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:52:19,744 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:52:19,916 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:52:20,106 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:52:20,157 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:52:20,323 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:52:20,372 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:52:20,502 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:52:20,552 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:52:20,742 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:52:20,792 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:52:20,905 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:52:20,955 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:52:21,068 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:52:21,118 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:52:21,319 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:52:21,369 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:52:21,542 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:52:21,609 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:52:21,885 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:52:21,936 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:52:22,068 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:53:20,029 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:54:20,284 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:55:20,539 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:55:20,719 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:55:20,914 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 17:56:20,799 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:56:20,979 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:56:21,171 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 17:56:21,221 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:56:21,372 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 17:56:21,422 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:56:21,554 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 17:56:21,604 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:56:21,848 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 17:56:21,900 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:56:22,015 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 17:56:22,079 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:56:22,198 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 17:56:22,252 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:56:22,440 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 17:56:22,490 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:56:22,656 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 17:56:22,706 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:56:22,879 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 17:56:22,928 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:56:23,056 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 17:57:21,079 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:58:21,331 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 17:59:21,572 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:00:21,811 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:00:21,930 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.shift_type.shift_type.process_auto_attendance_for_all_shifts for erpnext.ap.com
2023-05-22 18:00:22,162 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.shift_type.shift_type.process_auto_attendance_for_all_shifts for erpnext.ap.com
2023-05-22 18:00:22,221 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.connectors.shopify_connection.sync_old_orders for erpnext.ap.com
2023-05-22 18:00:22,495 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.connectors.shopify_connection.sync_old_orders for erpnext.ap.com
2023-05-22 18:00:22,556 INFO scheduler Scheduled Job Start: erpnext.support.doctype.issue.issue.set_service_level_agreement_variance for erpnext.ap.com
2023-05-22 18:00:22,781 INFO scheduler Scheduled Job Complete: erpnext.support.doctype.issue.issue.set_service_level_agreement_variance for erpnext.ap.com
2023-05-22 18:00:22,836 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.collect_project_status for erpnext.ap.com
2023-05-22 18:00:23,069 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.collect_project_status for erpnext.ap.com
2023-05-22 18:00:23,129 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.hourly_reminder for erpnext.ap.com
2023-05-22 18:00:23,372 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.hourly_reminder for erpnext.ap.com
2023-05-22 18:00:23,427 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.automatic_synchronization for erpnext.ap.com
2023-05-22 18:00:23,684 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.automatic_synchronization for erpnext.ap.com
2023-05-22 18:00:23,739 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.doctype.amazon_mws_settings.amazon_mws_settings.schedule_get_order_details for erpnext.ap.com
2023-05-22 18:00:23,921 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.doctype.amazon_mws_settings.amazon_mws_settings.schedule_get_order_details for erpnext.ap.com
2023-05-22 18:00:23,978 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.trigger_emails for erpnext.ap.com
2023-05-22 18:00:24,159 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.trigger_emails for erpnext.ap.com
2023-05-22 18:00:24,216 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:00:24,376 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:00:24,426 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:00:24,580 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:00:24,631 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:00:24,770 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:00:24,821 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:00:25,017 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:00:25,069 INFO scheduler Scheduled Job Start: erpnext.utilities.doctype.video.video.update_youtube_data for erpnext.ap.com
2023-05-22 18:00:25,305 INFO scheduler Scheduled Job Complete: erpnext.utilities.doctype.video.video.update_youtube_data for erpnext.ap.com
2023-05-22 18:00:25,361 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:00:25,554 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:00:25,608 INFO scheduler Scheduled Job Start: frappe.email.doctype.newsletter.newsletter.send_scheduled_email for erpnext.ap.com
2023-05-22 18:00:25,830 INFO scheduler Scheduled Job Complete: frappe.email.doctype.newsletter.newsletter.send_scheduled_email for erpnext.ap.com
2023-05-22 18:00:25,884 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.google_calendar.google_calendar.sync for erpnext.ap.com
2023-05-22 18:00:26,195 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.google_calendar.google_calendar.sync for erpnext.ap.com
2023-05-22 18:00:26,251 INFO scheduler Scheduled Job Start: frappe.desk.form.document_follow.send_hourly_updates for erpnext.ap.com
2023-05-22 18:00:26,459 INFO scheduler Scheduled Job Complete: frappe.desk.form.document_follow.send_hourly_updates for erpnext.ap.com
2023-05-22 18:00:26,513 INFO scheduler Scheduled Job Start: frappe.deferred_insert.save_to_db for erpnext.ap.com
2023-05-22 18:00:26,756 INFO scheduler Scheduled Job Complete: frappe.deferred_insert.save_to_db for erpnext.ap.com
2023-05-22 18:00:26,810 INFO scheduler Scheduled Job Start: frappe.desk.page.backups.backups.delete_downloadable_backups for erpnext.ap.com
2023-05-22 18:00:27,021 INFO scheduler Scheduled Job Complete: frappe.desk.page.backups.backups.delete_downloadable_backups for erpnext.ap.com
2023-05-22 18:00:27,075 INFO scheduler Scheduled Job Start: frappe.utils.error.collect_error_snapshots for erpnext.ap.com
2023-05-22 18:00:27,309 INFO scheduler Scheduled Job Complete: frappe.utils.error.collect_error_snapshots for erpnext.ap.com
2023-05-22 18:00:27,364 INFO scheduler Scheduled Job Start: frappe.model.utils.user_settings.sync_user_settings for erpnext.ap.com
2023-05-22 18:00:27,540 INFO scheduler Scheduled Job Complete: frappe.model.utils.user_settings.sync_user_settings for erpnext.ap.com
2023-05-22 18:00:27,593 INFO scheduler Scheduled Job Start: frappe.model.utils.link_count.update_link_count for erpnext.ap.com
2023-05-22 18:00:27,773 INFO scheduler Scheduled Job Complete: frappe.model.utils.link_count.update_link_count for erpnext.ap.com
2023-05-22 18:00:27,838 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:00:27,953 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:00:28,014 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:00:28,133 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:00:28,194 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:00:28,397 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:00:28,451 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:00:28,614 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:00:28,664 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:00:28,842 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:00:28,893 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:00:29,020 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:00:29,072 INFO scheduler Scheduled Job Start: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 18:00:29,281 INFO scheduler Scheduled Job Complete: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 18:00:29,337 INFO scheduler Scheduled Job Start: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 18:00:29,565 INFO scheduler Scheduled Job Complete: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 18:00:29,621 INFO scheduler Scheduled Job Start: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 18:00:29,827 INFO scheduler Scheduled Job Complete: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 18:00:29,893 INFO scheduler Scheduled Job Start: erpnext.stock.doctype.repost_item_valuation.repost_item_valuation.repost_entries for erpnext.ap.com
2023-05-22 18:00:30,077 INFO scheduler Scheduled Job Complete: erpnext.stock.doctype.repost_item_valuation.repost_item_valuation.repost_entries for erpnext.ap.com
2023-05-22 18:00:30,133 INFO scheduler Scheduled Job Start: erpnext.accounts.doctype.subscription.subscription.process_all for erpnext.ap.com
2023-05-22 18:00:30,357 INFO scheduler Scheduled Job Complete: erpnext.accounts.doctype.subscription.subscription.process_all for erpnext.ap.com
2023-05-22 18:01:22,190 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:02:22,416 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:03:22,647 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:04:22,892 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:04:23,055 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:04:23,240 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:04:23,300 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:04:23,483 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:04:23,540 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:04:23,692 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:04:23,743 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:04:23,963 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:04:24,019 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:04:24,143 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:04:24,208 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:04:24,325 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:04:24,379 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:04:24,591 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:04:24,655 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:04:24,861 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:04:24,922 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:04:25,116 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:04:25,167 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:04:25,296 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:05:23,162 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:05:23,325 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:05:23,528 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:06:23,410 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:07:23,704 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:08:23,948 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:08:24,109 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:08:24,279 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:08:24,328 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:08:24,474 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:08:24,523 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:08:24,663 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:08:24,712 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:08:24,893 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:08:24,943 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:08:25,055 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:08:25,105 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:08:25,217 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:08:25,279 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:08:25,461 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:08:25,510 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:08:25,682 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:08:25,732 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:08:25,905 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:08:25,954 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:08:26,090 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:09:24,218 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:10:24,473 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:10:24,647 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:10:24,846 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:11:24,725 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:12:24,972 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:12:25,131 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:12:25,334 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:12:25,384 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:12:25,531 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:12:25,581 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:12:25,709 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:12:25,759 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:12:25,941 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:12:25,992 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:12:26,101 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:12:26,154 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:12:26,279 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:12:26,328 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:12:26,510 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:12:26,560 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:12:26,720 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:12:26,770 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:12:26,944 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:12:26,995 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:12:27,121 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:13:25,247 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:14:25,542 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:15:25,796 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:15:25,978 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:15:26,170 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:15:26,233 INFO scheduler Scheduled Job Start: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 18:15:26,462 INFO scheduler Scheduled Job Complete: frappe.twofactor.delete_all_barcodes_for_users for erpnext.ap.com
2023-05-22 18:15:26,518 INFO scheduler Scheduled Job Start: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 18:15:26,782 INFO scheduler Scheduled Job Complete: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext.ap.com
2023-05-22 18:15:26,836 INFO scheduler Scheduled Job Start: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 18:15:27,053 INFO scheduler Scheduled Job Complete: frappe.oauth.delete_oauth2_data for erpnext.ap.com
2023-05-22 18:16:26,055 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:16:26,233 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:16:26,395 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:16:26,446 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:16:26,595 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:16:26,645 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:16:26,774 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:16:26,824 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:16:27,041 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:16:27,096 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:16:27,210 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:16:27,260 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:16:27,377 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:16:27,428 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:16:27,619 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:16:27,669 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:16:27,848 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:16:27,900 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:16:28,105 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:16:28,164 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:16:28,318 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:17:26,346 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:18:26,621 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:19:26,857 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:20:27,120 ERROR scheduler Exception in Enqueue Events for Site MyTrialERP.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 684, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 161, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 85, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 92, in get_connection
    conn.select_db(self.user)
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 496, in select_db
    self._read_ok_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 443, in _read_ok_packet
    pkt = self._read_packet()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "env/lib/python3.8/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "env/lib/python3.8/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1049, "Unknown database '_c296b757cb15d301'")

2023-05-22 18:20:27,283 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:20:27,487 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext.ap.com
2023-05-22 18:20:27,551 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:20:27,747 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext.ap.com
2023-05-22 18:20:27,800 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:20:27,933 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext.ap.com
2023-05-22 18:20:27,983 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:20:28,188 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext.ap.com
2023-05-22 18:20:28,249 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:20:28,443 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_log.bom_update_log.resume_bom_cost_update_jobs for erpnext.ap.com
2023-05-22 18:20:28,511 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:20:28,633 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext.ap.com
2023-05-22 18:20:28,686 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:20:28,801 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext.ap.com
2023-05-22 18:20:28,852 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:20:29,046 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext.ap.com
2023-05-22 18:20:29,097 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:20:29,274 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext.ap.com
2023-05-22 18:20:29,324 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:20:29,524 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext.ap.com
2023-05-22 18:20:29,574 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext.ap.com
2023-05-22 18:20:29,702 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext.ap.com
